docs/admin_console/event_integrations/relationships.dsl

```diff
@@ -0,0 +1,59 @@
+# Top Level event publishing
+server.api -> server.azure_service_bus.event_topic "Sends events to"
+server.events -> server.azure_service_bus.event_topic "Sends events to"
+server.api -> server.rabbit_mq.event_exchange "Sends events to"
+server.events -> server.rabbit_mq.event_exchange "Sends events to"
+
+# Azure Service Bus topics, subscriptions, and routing
+server.azure_service_bus.event_topic -> server.azure_service_bus.eventsWriteSub  "Fan-out Subscription"
+server.azure_service_bus.event_topic -> server.azure_service_bus.eventsHecSub "Fan-out Subscription"
+server.azure_service_bus.event_topic -> server.azure_service_bus.eventsWebhookSub  "Fan-out Subscription"
+server.azure_service_bus.event_topic -> server.azure_service_bus.eventsSlackSub "Fan-out Subscription"
```

Comment:
`Fan-out subscription` sounds more like a technology than a relationship description to me